pkg/ast/sourceStmt.go

```diff
@@ -54,6 +54,8 @@ type JsonStreamField struct {
 	Type       string                      `json:"type"`
 	Items      *JsonStreamField            `json:"items,omitempty"`
 	Properties map[string]*JsonStreamField `json:"properties,omitempty"`
+
+	Selected bool
```

Comment:
This looks like a state. Is this used as a singleton? Otherwise, the shared state may be affected by each other.